napari/benchmarks/benchmark_tracks_layer.py

```diff
@@ -0,0 +1,38 @@
+import numpy as np
+
+from napari.layers import Tracks
+
+
+class TracksSuite:
+    param_names = ['size', 'n_tracks']
+    params = [(5 * np.power(10, np.arange(2, 7))).tolist(), [10, 100, 1000]]
```

Comment:
I think it would be worth including 5, and 50 to cover the smaller number of tracks. Should be fast to run and gives us a little more information.

```suggestion
    params = [(5 * np.power(10, np.arange(7))).tolist(), [10, 100, 1000]]
```